site/content/en/docs/Examples/supertuxkart.md

```diff
@@ -0,0 +1,94 @@
+## Create a Fleet
+
+Let's create a Fleet using the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/examples/supertuxkart/fleet.yaml
+```
+
+You should see a successful output similar to this :
+
+```
+fleet.agones.dev/supertuxkart created
+```
+
+This has created a Fleet record inside Kubernetes, which in turn creates two warm [GameServers]({{< ref "/docs/Reference/gameserver.md" >}})
+that are available to be allocated for a game session.
+
+```bash
+kubectl get fleet
+```
+It should look something like this:
+
+```
+NAME           SCHEDULING   DESIRED   CURRENT   ALLOCATED   READY   AGE
+supertuxkart   Packed       2         2         0           2       55s
+```
+
+You can also see the GameServers that have been created by the Fleet by running `kubectl get gameservers`,
+the GameServer will be prefixed by `supertuxkart`.
+
+```
+NAME                             STATE       ADDRESS        PORT   NODE                                  AGE
+supertuxkart-xfw2g-bwwkb         Ready       34.82.158.69   7421   gke-agon-default-pool-f18c8e90-1f9k   103s
+supertuxkart-xfw2g-skdnf         Ready       34.82.158.69   7585   gke-agon-default-pool-f18c8e90-1f9k   103s
+```
+
+For the full details of the YAML file head to the [Fleet Specification Guide]({{< ref "/docs/Reference/fleet.md" >}})
+
+{{< alert title="Note" color="info">}} The game servers deployed from a `Fleet` resource will be deployed in the same namespace. The above example omits specifying a namespace, which implies both the `Fleet` and the associated `GameServer` resources will be deployed to the `default` namespace. {{< /alert >}}
+
+## Viewing GameServer Logs
+
+For troubleshooting or to check how your game servers are running, you can look at the logs of a specific pod using:
+
+```bash
+kubectl logs -f <supertuxkart-game-server-pod-name>
+```
+
+## Connect to the Game Server
+
+After allocating a GameServer from the fleet and obtaining its status and IP, you're ready to connect and play. Here’s how to use the server IP and port to join the game with the SuperTuxKart client:
+
+**Launch SuperTuxKart**: Navigate to the SuperTuxKart client you downloaded earlier. Execute the `run_game.sh` script to launch and play the game.
+
+**Navigate to Online Play**: From the main menu, select the "Online" option and then select "Enter server address" from the available options.
+
+**Enter Server Details**: In the subsequent screen, you will be prompted to input the IP address and port number in order to join the game. Please enter the IP address and port number obtained from the `kubectl get gameservers` command.
+
+![enter ip and port](../../../images/supertuxkart-enter-ip-port.png)
+
+**Join the Game**: After entering the server details, proceed to join the server. You should now be connected to your Agones-managed SuperTuxKart game server and ready to play.
+
+![start race](../../../images/supertuxkart-start-race.png)
```

Comment:
We can also connect some AI bots!

I do a demo of this here:
https://youtu.be/ACJvwZBwWew?list=PLqqp1QEhKwa60zuzZej8-va3Q_Za6bchq&t=1027

Documentation here: https://github.com/supertuxkart/stk-code/blob/master/NETWORKING.md#testing-server

So you can have other players - can we please add this too!

Reply:
Information on playing with AI bots has been included. Please take a moment to review it and share your thoughts.